scenariolib/event_click.go

```diff
@@ -65,11 +65,11 @@ func newClickEvent(e *JSONEvent) (*ClickEvent, error) {
 		if event.fakeClick, validcast = e.Arguments["fakeClick"].(bool); !validcast {
 			return nil, errors.New("Parameter fakeClick must be a boolean value")
 		}
-		if e.Arguments["falseResponse"] != nil {
-			jsonFalseResponse, _ := json.Marshal(e.Arguments["falseResponse"])
-			err := json.Unmarshal(jsonFalseResponse, &event.fakeResponse)
+		if e.Arguments["fakeResponse"] != nil {
```

Comment:
I could print a warning message, but still handle it.
What do you think?
`Parameter falseResponse is deprecated, please use fakeResponse instead.`

And still handle `falseResponse` anyway?